src/Keboola/DbExtractor/Application.php

```diff
@@ -118,9 +128,9 @@ private function runAction(): array
     {
```

Comment:
navrh: co udelat *action metody public - jsou to vnejsi entrypointy (i kdyz se volaji dynamicky zevnitr) a maji dany interface, takhle jsou mylne oznacene jako private/unused a vypada to, ze se daji libovolne odstranit/zmenit. predpokladam, ze si na to bude stezovat codeclimate pokud ho nekdy zavedeme

Reply:
I kdyz je dame jako public tak nic nevyresime, protoze se volaji takto
https://github.com/keboola/db-extractor-common/blob/master/src/Keboola/DbExtractor/Application.php#L65

Nechal bych to uz jak to je. V dalsim PR se to volani sice meni a nebude dynamicke:
https://github.com/keboola/db-extractor-common/blob/david-adapter/src/Keboola/DbExtractor/Extractor/ExtractorAdapter.php#L27

Nicmene stejne bych ty metody nedaval public - odkud je chces volat nez, pres `run.php`? To, ze se to cele spusti zaobalene ti hlavne zarucuje napr. validni config.

Reply:
ok, pokud to nebude dynamicke, tak  tim asi neni potreba nic delat